packages/devextreme/js/ui/popup/ui.popup.js

```diff
@@ -510,7 +518,9 @@ const Popup = Overlay.inherit({
             toolbarsItems.push(this._getCloseButton());
         }
 
-        return toolbarsItems;
+        const sortedActionItems = sortActionButtonsItems(doneCancelButtonsInfo).map(item => item.item);
+
+        return toolbarsItems.concat(...sortedActionItems);
```

Comment:
So, cancel and apply buttons are always placed in the end now? I am not sure about this because user should have an opportunity to specify the order. I thought that we should change only the default order.